src/components/ImportForm/GitImportForm.tsx

```diff
@@ -149,13 +154,16 @@ const GitImportForm: React.FunctionComponent<GitImportFormProps> = ({
       // eslint-disable-next-line camelcase
       step_name: 'review-step',
     });
-    setReviewMode(false);
+    showModal<{ leave: boolean }>(createCloseImportFormModal()).closed.then(({ leave }) => {
+      if (leave) {
+        setReviewMode(false);
+      }
+    });
   };
 
   return (
     <Formik
       onSubmit={reviewMode ? handleSubmit : handleNext}
-      onReset={handleReset}
```

Comment:
Why change sending reset handler from formik to directly?

Reply:
because it would have reset the values of form before confirmation from the modal.